src/shared/components/case-viewer/case-event-trigger/case-event-trigger.component.ts

```diff
@@ -29,9 +29,15 @@ export class CaseEventTriggerComponent implements OnInit, OnDestroy {
     private route: ActivatedRoute,
     private caseReferencePipe: CaseReferencePipe,
     private activityPollingService: ActivityPollingService
-  ) { }
+  ) {
+    // Doesn't look like this is ever called so this may be a redundant component now...
+    // It may, of course, just be that I don't know how to get to this component.
+    console.log('CaseEvenTriggerComponent constructor');
+  }
 
   ngOnInit(): void {
+    // Doesn't look like this is ever called so this may be a redundant component now...
+    console.log('CaseEvenTriggerComponent.ngOnInit', this.route.snapshot.data);
```

Comment:
please remove console.log

Reply:
This is a draft. All the console logs will go. There's a comment above it to explain why it's there. :)